server/boot/index.js

```diff
@@ -38,6 +39,23 @@ function setup() {
 
 		// setup logger
 		app.use( morgan( 'dev' ) );
+
+		if ( config.isEnabled( 'wpcom-user-bootstrap' ) ) {
+			if ( config( 'wordpress_logged_in_cookie' ) ) {
+				const username = config( 'wordpress_logged_in_cookie' ).split( '%7C' )[ 0 ];
+				console.info( chalk.cyan( '\nYour logged in cookie set to user: ' + username ) );
+
+				app.use( function( req, res, next ) {
+					if ( ! req.cookies.wordpress_logged_in ) {
+						req.cookies.wordpress_logged_in = config( 'wordpress_logged_in_cookie' );
+					}
+					next();
+				} );
+			} else {
+				console.info( chalk.cyan( '\nYou need to set `wordpress_logged_in_cookie` in secrets.json' +
+					' for wpcom-user-bootstrap to work in development.' ) );
```

Comment:
Should we display this error message in orange (or red) to make it more prominent?